cmd/piper.go

```diff
@@ -18,16 +18,17 @@ import (
 
 // GeneralConfigOptions contains all global configuration options for piper binary
 type GeneralConfigOptions struct {
-	CustomConfig   string
-	DefaultConfig  []string //ordered list of Piper default configurations. Can be filePath or ENV containing JSON in format 'ENV:MY_ENV_VAR'
-	ParametersJSON string
-	EnvRootPath    string
-	NoTelemetry    bool
-	StageName      string
-	StepConfigJSON string
-	StepMetadata   string //metadata to be considered, can be filePath or ENV containing JSON in format 'ENV:MY_ENV_VAR'
-	StepName       string
-	Verbose        bool
+	CustomConfig         string
+	DefaultConfig        []string //ordered list of Piper default configurations. Can be filePath or ENV containing JSON in format 'ENV:MY_ENV_VAR'
+	ParametersJSON       string
+	EnvRootPath          string
+	NoTelemetry          bool
+	StageName            string
+	StepConfigJSON       string
+	StepMetadata         string //metadata to be considered, can be filePath or ENV containing JSON in format 'ENV:MY_ENV_VAR'
+	StepName             string
+	Verbose              bool
+	MessageOnlyLogFormat bool
```

Comment:
@daniel-kurzynski maybe we should change this to a string parameter so we can add more formatters in the future?
something like `--logformat=plaintext`?